operator-metadata/image.yaml

```diff
@@ -0,0 +1,47 @@
+schema_version: 1
+
+name: amq7/amq-streams-operator-metadata
+description: "AMQ Streams image containing the operator metadata"
+version: "1.3.0"
+from: ubi7-minimal:7-released
+
+labels:
+  - name: "com.redhat.component"
+    value: "amqstreams-operator-metadata-container"
+  - name: "io.k8s.description"
+    value: "Operator metadata"
+  - name: "io.k8s.display-name"
+    value: "AMQ Streams Operator metadata"
+  - name: "io.openshift.tags"
+    value: "messaging,amq,jboss"
+  - name: "licenses"
+    value: "/root/licenses"
```

Comment:
Should be safe to remove the licenses label

Reply:
Where and how can we sure of what are the labels which make sense for this image?

Reply:
We will know once the RCM requirements are fleshed out